### DISCUSSION TITLE
Google Calendar as a JSON data source

### DISCUSSION THREAD

**Discussion by lewiskerns** [code4recovery/12-step-meeting-list#1143] [Q&A] [answered]
Is it possible to use a Google calendar as a JSON data source to automatically pull in meetings to TSML?  A district I am supporting has been manually entering meetings into the plugin but would like to switch to using a Google Calendar for their data entry.  If this is still possible, does anyone have any instructions on obtaining the google link?

**Comment by kiyote33** [comment, marked as answer]
Hi @lewiskerns, I don't believe we support pulling a Google calendar json file directly, but we do have support for Importing Google Sheets. You can see documentation on our website at [https://sheets.code4recovery.org/](https://sheets.code4recovery.org/). Let us know if this is what your looking for.
